d2format/format.go

```diff
@@ -409,3 +409,11 @@ func KeyPath(kp *d2ast.KeyPath) (ida []string) {
 	}
 	return ida
 }
+
+func IDA(ida []string) *d2ast.KeyPath {
```

Comment:
Nice though perhaps this should be in d2ast instead?

Reply:
yeah want me to get the one above this too? it's just a mirror

Reply:
Nah, that is d2format specific. Oh lmfao we already have IDA in d2ast, it's called `d2ast.MakeKeyPath`.

Reply:
damnit i thought there might be